hw2/02-reverse.js

```diff
@@ -1,3 +1,13 @@
 /** Exercise 02 - Reverse **/
 
 // Add your code here
+function reverse() {
+  const Userinput = document.getElementById("input").value;
+  const backward = Userinput.split("").reverse().join("");
+  if (backward.toString().length !== 8)
+    document.getElementById("display").innerHTML =
+      "Error: Please input an 8-digit number";
+  else
+    document.getElementById("display").innerHTML =
+      Userinput + " --> " + backward;
```

Comment:
Missing text color styling. Green text for a successful message and red for an error